src/GADM.jl

```diff
@@ -122,4 +123,25 @@ function get(country, subregions...)
     throw(ArgumentError("feature not found"))
 end
 
+"""
+    coordinates(country, levels...)  
+
+Returns a deep array of Float64 coordinates for the requested region.  
```

Comment:
I would remove `Float64` from the docstring as it is something low-level that someone reading a docstring usually isn't interested in knowing.